### DISCUSSION TITLE
`SemanticWebImport` Plugin for Gephi 0.10.0

### DISCUSSION THREAD

**Discussion by tbohne** [gephi/gephi#2732] [Q&A]
I installed the Linux version of `Gephi 0.10.0` and am not able to use the `SemanticWebImport` plugin. It's neither part of the "Available Plugins" nor can it be installed manually. I downloaded it (.nbm), but when I try to install it using the "Plugin Installer", I get `The plugin org.gephi.io.processor.plugin is requested in version 0.9.2.`, which I'm also not able to install for some reason.

Has the plugin `SemanticWebImport` been renamed or does anyone have some guidance for this problem?

Kind regards,
Tim

*Update*:
I also installed `Gephi 0.9.2` - there I can install the `SemanticWebImport` plugin from the catalogue, but when I try to visualize an RDF file (.ttl), I get the following error: `INFO: java.lang.NoClassDefFoundError: Could not initialize class com.ibm.icu.lang.UCharacter`.

**Comment by mbastian** [comment]
Hi @tbohne , that plugin has not been updated for 0.9.3+ versions (see https://github.com/gephi/gephi-plugins/issues/280). It won't work on Gephi 0.10.0. It may have been broken in previous versions but I'm not sure.
